### PR TITLE
BL-365 [FRONT END] patch/instructor_newsfeed/sort_posts_newest_to_oldest

### PR DESCRIPTION
**Pull request by anastasia-lapteva**
## Description

Jira Ticket # **BL-365** Patch: InstructorNewsFeed currently lists posts from oldest to newest.

Co-Author: **Vasilii Garanin**

## Fixes

InstructorNewsFeed now displays the most recent posts at the top of the web browser.

## Loom Video

https://www.loom.com/share/78515bca4dd14a8b9e9b60080e0d1d92

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by AnthonyLayne — APPROVED** [review]
Nice job figuring that out, your code looks good, didn't see any improvements that needed to be made. Great Job!

**Review by gclancy121 — COMMENTED** [review]
looks good to me

**Review by Rlauckern — APPROVED** [review]
Good and clean code. It's much nicer being posted at the top!

**Review by Roadlyfe — APPROVED** [review]
Excellent work! clean code and easy to follow the steps taken.

**Review by reneetro — APPROVED** [review]
Looks good!

**Review by Rebecca-Stone — APPROVED** [review]
These changes look great, everything looks nice and dry. I liked your variable name, easy for future developers to know what they are using/looking at. Great work on this!

**Review by tkordysh — APPROVED** [review]
Changes look good to me, and the order makes more sense now. Thanks for this!

File: src/redux/reducers/instructorReducer.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const initialState = {
   post: [],
   postOptions: 'newsFeed',
   postID: 0,
+  posted_at: null,
```
Comment (ryan-hamblin) [inline src/redux/reducers/instructorReducer.js:35]: only hesitation I have with a 'null' case here is that your API _may not_ be set up to receive a null in the instance where this ID could be type `str` or `int/Number`

**Review by ryan-hamblin — COMMENTED** [review]
The work here definitely follows the redux/react-redux standard and desired path. There were a few things I noticed that I'd love to have further conversations about. However, this work was done cleanly and adds the value that it set out to add.

**Review by Cassius-Cassity — APPROVED** [review]
Really like the use of the spreader operator on the handle edit! It just looks clean compared to writing each item out.

**Pull request merged**